default/python/tox.ini

```diff
@@ -4,6 +4,7 @@ exclude =
   AI/
 putty-ignore =
   *.py : +E121,E123,E126,E226,E24,E704,W503 # default ignore list
+  *.py : +F812 # list comprehension redefines '<variable>' from line <line_no>
```

Comment:
I'd strongly prefer to keep that rule active. This is more than just a code-style issue and may introduce hard to find bugs.
```
foo = 'test'
bar = [foo for foo in range(3)]
print foo  # oops it's 2 now!
```

Reply:
I plan to fix it later, after this check will be included to the CI  run.